src/GeocoderJsonConsumer.php

```diff
@@ -1,13 +1,10 @@
 <?php
-/**
- * @file
- * Contains /Drupal/geocoder_autocomplete/GeocoderJsonConsumer.
- */
 
 namespace Drupal\geocoder_autocomplete;
 
-use GuzzleHttp\ClientInterface;
 use Drupal\Core\Language\LanguageManagerInterface;
+use Drupal\Component\Utility\Html;
```

Comment:
It is used in line 77 and 78 of the file:
https://github.com/itamair/geocoder_autocomplete/pull/1/commits/ae5a4d0247ad8bdb0138db9ab75337714c68d57d